utils/utils.go

```diff
@@ -41,6 +41,13 @@ func MinInt(a, b uint32) uint32 {
 	return b
 }
 
```

Comment:
If `MinInt` is not used, we can remove it. If it is, I guess it's better to rename it to `MinInt32`.

Reply:
Yeah. MinInt still being used. I'll rename it to 32 then

Reply:
Done :+1: